Straight_skeleton_2/doc/Straight_skeleton_2/Concepts/StraightSkeletonBuilderTraits_2.h

```diff
@@ -138,9 +158,15 @@ typedef unspecified_type Is_edge_facing_ss_node_2;
 /*!
 A predicate object type.
 
-Must provide `CGAL::Oriented_side operator()( const Trisegment_2_ptr& e, const Segment_2& e0, const Segment_2& e1, const Trisegment_2_ptr& e01_event, bool e0_is_primary) const`,
+Must provide
+
+`CGAL::Oriented_side operator()( const Trisegment_2_ptr& e,
+                                 const Segment_2& e0, const FT& w0,
+                                 const Segment_2& e1, const FT& w1,
```

Comment:
During overnight documentation builds we get:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-215/doc/Straight_skeleton_2/Concepts/StraightSkeletonBuilderTraits_2.h:162: warning: explicit link request to 'Oriented_side' could not be resolved
```
Might it be that `Segment_2` should be `Segment_2_with_ID`?

Reply:
`Segment_2_with_ID` is a type specific to CGAL's implementation of the concept. In the model, it is the typedef `Segment_2`, and the concept does use `Segment_2`.

What made you think of this since the warning refers to the result type? I must say I'm a little confused by the warning given that there is no change from master except a line break such that the `operator()` is not on the same line as `Must provide...`.

Reply:
Indeed looks like the remark from me does not cover the problem. You pointed in the right direction the problem  looks like to be the fact that the `` ` ... ` `` part is split over **more** than 2 lines (it looks like one `\n` is allowed but no more.

It is possible but to split over more than 2 lines but one one has to use `` <code> ... </code> `` instead of `` ` ... ` ``.

Reply:
As it is an intriguing problem I looked in the doxygen code and I see here:
```
if (nl==2) // too many newlines inside the span
{
  return 0;
}
```
(so 2 or more newlines are not allowed, probably to overcome slow processing in case the closing `` ` `` is missing).

Reply:
Thanks for debugging it.

Reply:
The doxygen code has been integrated in the master on GitHub.